client/src/modules/vouchers/toolkit/payment_employee/payment_employee.modal.html

```diff
@@ -0,0 +1,58 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static" translate>TREE.FINANCE</li>
+    <li class="title" translate>VOUCHERS.GLOBAL.PAYMENT_EMPLOYEES</li>
+  </ol>
+</div>
+
+<form name="ToolForm" bh-submit="ToolCtrl.import(ToolForm)" novalidate>
+  <div class="modal-body">
+    <!-- cashbox -->
+    <bh-cashbox-select
+      cashbox-id = "ToolCtrl.searchQueries.cashbox_id"
+      name = "cashbox_id"
+      on-select-callback = "ToolCtrl.onSelectCashbox(cashbox)">
+    </bh-cashbox-select>
+
+    <bh-payroll-period-select
+      id="payroll_configuration_id"
+      payroll-configuration-id="ToolCtrl.searchQueries.payroll_configuration_id"
+      name="PayrollPeriodForm"
+      on-select-callback="ToolCtrl.onSelectPayrollPeriod(payrollConfig)"
+      required="true"
+      validation-trigger="ToolForm.$submitted">
+    </bh-payroll-period-select>
+ 
+    <!-- select Employees Paiements  -->
+    <div ng-show="ToolCtrl.gridDisplay">
+      <div
+        id="paymentGrid"
+        style="height: 40vh; margin-bottom: 20px;"
+        ui-grid="ToolCtrl.gridOptions"
+        ui-grid-filtering
+        ui-grid-auto-resize
+        ui-grid-selection>
+      </div>
+    </div>
+
+    <div ng-show="ToolCtrl.gridDisplay" class="text-right">
+      <span translate>FORM.LABELS.TOTAL</span>: <strong class="text-danger">{{ ToolCtrl.totalNetSalary || 0 | currency: ToolCtrl.currencyId }}</strong><br>
```

Comment:
In the future, these would be easier to read with parentheses:
```html
<strong class="text-danger">{{ (ToolCtrl.totalNetSalary || 0) | currency: ToolCtrl.currencyId }}</strong/
```
It just makes it easier to parse.